langserver/langserver_test.go

```diff
@@ -30,6 +30,7 @@ import (
 
 type serverTestCase struct {
 	skip    bool
+	focus   bool
```

Comment:
go supports this already since each test case is a subtest. You just need to do `go test -run TestServer/focusregex`

Reply:
I basically just used it to focus the tests in code so that I can easily debug the tests using https://github.com/derekparker/delve (which btw works perfectly without any problems!)
I will revert it :)